commit fc62a94721d74e866bb99cf0544b6be2f90c2d2e
Author: Matt Flax <flatmax@flatmax.org>
Date:   Thu Aug 30 09:38:01 2018 +1000

    ASoC: cs4265: Add native 32bit I2S transport
    
    commit be47e75eb1419ffc1d9c26230963fd5fa3055097 upstream.
    
    The cs4265 uses 32 bit transport on the I2S bus. This patch enables native
    32 bit mode for machine drivers which use this sound card driver.
    
    Signed-off-by: Matt Flax <flatmax@flatmax.org>
    Reviewed-by: Charles Keepax <ckeepax@opensource.cirrus.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/codecs/cs4265.c b/sound/soc/codecs/cs4265.c
index c90959d7adf6..6d0843386948 100644
--- a/sound/soc/codecs/cs4265.c
+++ b/sound/soc/codecs/cs4265.c
@@ -497,7 +497,8 @@ static int cs4265_set_bias_level(struct snd_soc_component *component,
 			SNDRV_PCM_RATE_176400 | SNDRV_PCM_RATE_192000)
 
 #define CS4265_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_U16_LE | \
-			SNDRV_PCM_FMTBIT_S24_LE | SNDRV_PCM_FMTBIT_U24_LE)
+			SNDRV_PCM_FMTBIT_S24_LE | SNDRV_PCM_FMTBIT_U24_LE | \
+			SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_U32_LE)
 
 static const struct snd_soc_dai_ops cs4265_ops = {
 	.hw_params	= cs4265_pcm_hw_params,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1309_fc62a94721d74e866bb99cf0544b6be2f90c2d2e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1309_fc62a94721d74e866bb99cf0544b6be2f90c2d2e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
